Dockerfile

```diff
@@ -1,8 +1,16 @@
-# Docker Definition for ElasticSearch Curator
+FROM python:3.6-alpine3.6 as builder
 
-FROM python:2.7.8-slim
-MAINTAINER Christian R. Vozar <christian@rogueethic.com>
+RUN apk --no-cache add build-base tar musl-utils openssl-dev
+RUN pip3 install setuptools cx_Freeze==6.0b1 requests-aws4auth
```

Comment:
You might want to also add `boto3` and `botocore` if someone is going to be using AWS Elasticsearch.  See [here](https://github.com/elastic/curator/blob/master/curator/utils.py#L847-L848)